### PR TITLE
Endpoint view for Paid Checkout Requests

### PR DESCRIPTION
**Pull request by tmirmota**

### CODE REVIEW

**Review by AndrewEBolt — APPROVED** [review]

**Comment by jlfama** [comment]
@tmirmota What is the purpose of this example?
As far as look and functionality, can you give me details on how to run this?

File: react-checkout/src/App.css [hunk 1]
```diff
@@ -41,3 +49,46 @@ input:-webkit-autofill {
...
 }
+
+@media (min-width: 1200px) {
+  .card-fixed {
+    position: fixed;
+  }
+}
+
+pre {
+  white-space: pre-wrap; /* css-3 */
```
Comment (jlfama) [inline react-checkout/src/App.css]: When using vendor prefixes in CSS, the non-vendor prefixed version should always come last.

File: react-checkout/src/components/Request.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        delete invoice.address_street
+        delete invoice.address_city
+        delete invoice.address_postalcode
+        delete invoice.address_region_id
+        delete invoice.address_country_id
+        delete invoice.phone
+        delete invoice.created
+      }
+      if (tickets) {
+        tickets.map(ticket => {
```
Comment (jlfama) [inline react-checkout/src/components/Request.jsx:51]: Be aware that writing ES6 creates browser incompatibility (unless you're using something to compile the jsx to es5 that I'm not aware of). 
Depending on target audience, that may be something to think about.

**Comment by jlfama** [comment]
@tmirmota Just took a look at this. Brilliant addition to the example.
Minor change for the CSS and it's good to go.

File: react-checkout/src/components/Request.jsx (hunk 2 above)
Comment (tmirmota) [reply react-checkout/src/components/Request.jsx:51]: Using babel to compile into browser-friendly js. https://babeljs.io/

Comment (jlfama) [reply react-checkout/src/components/Request.jsx:51]: @tmirmota Where do you define that? I was looking for mention of it but couldn't find anything.

File: react-checkout/src/App.css (hunk 1 above)
Comment (tmirmota) [reply react-checkout/src/App.css]: Just moved this

File: react-checkout/src/components/Request.jsx (hunk 2 above)
Comment (tmirmota) [reply react-checkout/src/components/Request.jsx:51]: The app is built with create-react-app which comes pre built with the configuration for babel and webpack. If you want to see the config for each, you can run `npm run eject`. Keep in mind that once you eject you can not go back

Comment (tmirmota) [reply react-checkout/src/components/Request.jsx:51]: See docs on ejecting here: https://github.com/facebookincubator/create-react-app#converting-to-a-custom-setup

Comment (jlfama) [reply react-checkout/src/components/Request.jsx:51]: Thanks, all I needed to know was where the mystery box was

**Review by jlfama — APPROVED** [review]

**Pull request merged**